server/app/src/main/java/com/ams/app/dto/AttendanceReportdto.java

```diff
@@ -0,0 +1,12 @@
+package com.ams.app.dto;
+
+import lombok.Data;
+
+@Data
+public class AttendanceReportdto {
+    private int sid;
```

Comment:
Don't use primitive types much. It'll be good to use Wrapper Classes(Integer, Long..etc) as they'll store null values also

Reply:
Modified all Id's to Long and primitive type(int) to Wrapper class(Integer)